flagsmith/flagsmith.py

```diff
@@ -168,7 +173,9 @@ def get_identity_flags(
         return self._get_identity_flags_from_api(identifier, traits)
 
     def get_identity_segments(
-        self, identifier: str, traits: typing.Dict[str, typing.Any] = None
+        self,
+        identifier: str,
+        traits: typing.Optional[typing.Mapping[str, typing.Any]] = None,
```

Comment:
Wouldn't hurt to use the flag_engine.identities.traits.types.TraitValue type for the trait values here.